ros_cross_compile/docker/build_workspace.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
-set -euxo pipefail
+set -eo pipefail
```

Comment:
`-x` because it's unecessarily noisy for this script.

`-u` because `setup.bash` has unset variables. So, I added very explicit checks for the ones we care about